src/ontology/mondo-edit.obo

```diff
@@ -555743,6 +555743,7 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 [Term]
 id: MONDO:0859083
 name: systemic lupus erythematosus 17
+def: "An instance of systemic lupus erythematosus (disease) that is caused by mutations in TLR7." [https://orcid.org/0000-0003-1738-6117, OMIM:301080]
```

Comment:
@MeeSiing 

While the definition is not incorrect, we should follow the definition as defined in this pattern: [disease_series_by_gene](https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/disease_series_by_gene.yaml)

In addition, please add the following as cross-reference to the definition:
MONDO:patterns/disease_series_by_gene